scripts/monitr-server.py

```diff
@@ -24,5 +24,5 @@ def refilter_data_select(*events):
     site="labcore",
     title="data explorer",
     sidebar=[],
-    main=[ds, loader, loader.plot],
-).servable()
\ No newline at end of file
+    main=[ds, loader],
```

Comment:
why do we not need the loader.plot here anymore?

Reply:
Because I added the plot to the layout of loader. If loader.plot was still here, the plot would always be just an additional row at the end of the page, incorporating it into the layout of loader allows for a fixed height column to be placed next to it.